src/Mailer/Email.php

```diff
@@ -1830,6 +1841,29 @@ public function message($type = null)
     }
 
     /**
+     * Sets priority.
+     *
+     * @param int|null $priority 1 (highest) to 5 (lowest)
+     * @return $this
+     */
+    public function setPriority($priority)
+    {
+        $this->_priority = $priority;
```

Comment:
So far we never do too much validation here for those which are documented clearly enough.
The doc block reads "int" and `1 (highest) to 5 (lowest)`.
If you put something different in your email header, that is the developers fault then.
I don't think for example we validate the other headers like host etc.